extensions/kraken/exchange.js

```diff
@@ -0,0 +1,278 @@
+var KrakenClient = require('kraken-api'),
+  path = require('path'),
+  moment = require('moment'),
+  n = require('numbro'),
+  colors = require('colors')
+
+module.exports = function container(get, set, clear) {
+  var c = get('conf');
```

Comment:
I know there is not a real standardjs defined, but no other js files use `;`, would you please remove them from your changes?